stacker/environment.py

```diff
@@ -0,0 +1,11 @@
+
+
+def parse_environment(raw_environment):
+    environment = {}
+    for line in raw_environment.split('\n'):
+        if ':' not in line:
```

Comment:
Hmm - this is a little too loose.  You're trying to catch blank lines, right?

I'd maybe go with:

```
if not line:
    continue

try:
    key, value = line.split(":", 1)
except ValueError:
    raise Exception("Environment must be in key: value format")
```